code/src/components/Item.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+
+
+export const Item = ({ item }) => {
+  return (
+  <ItemContainer>{item.text}</ItemContainer>
+  )
+}
```

Comment:
I can't find we're you are using this component, because it's not imported somewhere (or I might miss that). But the code seems to work if you get some component styling to the ItemContainer.